examples/worlds/levels.sdf

```diff
@@ -2,7 +2,7 @@
 <!--
   Gazebo level manager demo
 
-  Start gazebo using the "--levels" flag.
+  Start gazebo using the "-/-levels" flag.
```

Comment:
Is this because xml doesn't like the double dashes? If so, we can wrap the whole comment in a CDATA block
see https://github.com/gazebosim/gz-sim/blob/2229f128e90ca1ce050e7e226a87a0efe1340adc/examples/worlds/shapes.sdf#L6-L10